README.md

```diff
@@ -22,6 +22,16 @@ typings search tape
 # Find an available definition (by name).
 typings search --name react
 
+# Install ambient typings (default from "DT", configurable through `defaultAmbientSource` in `.typingsrc`)
+typings install react --ambient --save
```

Comment:
Can we use this command only once. It's repeated below with a different comment.